Ex10.1.py

```diff
@@ -0,0 +1,47 @@
+import pandas
+import numpy
+import scipy
+import scipy.integrate as spint
+from plotnine import *
+r1=[-0.1,0.1,0.4,0.8,1]
+def mgrate(y,to,r1,K):
+    N=y[0]  
+    dNdt=r1* (1-N/K)* N
+    return [dNdt]
+for i in r1:
+    param1=(i,100)
+    N0=10
+    times=range(0,1000)
+ms1=spint.odeint(func=mgrate,y0=N0,t=times,args=param1)
+modelOutput=pandas.DataFrame({"t":times,"N":ms1[:,0]})
+mgraph=ggplot(modelOutput,aes(x="t",y="N"))+geom_line()+theme_classic()
+

```

Comment:
You want to put the following two lines in the loop to store the results
ms1=spint.odeint(func=mgrate,y0=N0,t=times,args=param1)
modelOutput=pandas.DataFrame({"t":times,"N":ms1[:,0]})

-0.25 pts